xarray/core/common.py

```diff
@@ -876,17 +881,16 @@ def rolling_exp(
 
         Parameters
         ----------
-        window : A single mapping from a dimension name to window value,
-                 optional
+        window : {dim: window_size}, optional
+            A single mapping from a dimension name to window value.
 
             dim : str
                 Name of the dimension to create the rolling exponential window
                 along (e.g., `time`).
             window : int
                 Size of the moving window. The type of this is specified in
                 `window_type`
-        window_type : str, one of ['span', 'com', 'halflife', 'alpha'],
-                      default 'span'
+        window_type : {"span", "com", "halflife", "alpha"}, default: "span"
             The format of the previously supplied window. Each is a simple
             numerical transformation of the others. Described in detail:
             https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.ewm.html
```

Comment:
```suggestion
            https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.ewm.html
```
the page seems to have moved